scripts/benchmark_db/Gemfile.lock

```diff
@@ -0,0 +1,34 @@
+GEM
+  remote: https://rubygems.org/
+  specs:
```

Comment:
I don't see logger in lock file.

Reply:
Fixed the issue with the bundler due to the system MacOS `ruby` installation, and was able to update the lockfile.